generators/entity-server/templates/src/main/java/package/web/rest/EntityResource.java.ejs

```diff
@@ -74,7 +73,8 @@ import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
   <%_ } else { _%>
 import org.springframework.web.util.UriComponentsBuilder;
   <%_ } _%>
-<%_ } else if (reactive) { _%>
+<%_ } _%>
+<%_ if (reactive || !paginationNo && (!(databaseTypeSql && reactive) || viaService)) { _%>
```

Comment:
```suggestion
<%_ if (reactive || !paginationNo) { _%>
```
`(!(databaseTypeSql && reactive) || viaService)` always resolves to true. Right?

Reply:
I don't know. The condition is not just `reactive` because this https://github.com/jhipster/generator-jhipster/blob/main/generators/entity-server/templates/src/main/java/package/common/get_all_stream_template.ejs

Reply:
Since reactive is always false at the removed condition, it would always resolve to true.
`!paginationNo` should be enough for non reactive condition.